vllm/engine/llm_engine.py

```diff
@@ -1236,8 +1236,9 @@ def _advance_to_next_step(
                     seq_group, seq_group_metadata,
                     seq_group.state.num_steps == 1)
             else:
-                seq_group.update_num_computed_tokens(
-                    seq_group_metadata.token_chunk_size)
+                token_chunk_size = (seq_group_metadata.token_chunk_size if
+                                    seq_group_metadata.token_chunk_size else 0)
```

Comment:
```suggestion
                                    seq_group_metadata.token_chunk_size is not None else 0)
```

Reply:
same result, just a style preference?

Reply:
It's to emphasize that `seq_group_metadata.token_chunk_size` can be `None` (since it's not clear from just reading on GitHub).